packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.module.scss

```diff
@@ -0,0 +1,23 @@
+:local {
+  .background-tile {
+    background-color: var(--terra-demographics-banner-background-tile-background-color, #014979);
+    background-image: var(--terra-demographics-banner-background-tile-background-image, linear-gradient(#014979 8%, #006caa 100%));
+    box-shadow: var(--terra-demographics-banner-background-tile-box-shadow, inset 0 -1px 2px 0 #014979);
+    display: flex;
+    flex-direction: row;
+    padding: var(--terra-demographics-banner-background-tile-padding, 0.21428rem 0.42857rem 0.28571rem);
```

Comment:
this is a ux concern, but i think it looks odd that the bottom padding is slightly larger than the top. it makes the title slightly shifted up